src/main/scala/nl.knaw.dans.easy.properties/app/legacyImport/ImportProps.scala

```diff
@@ -171,6 +163,20 @@ class ImportProps(repository: Repository, interactor: Interactor, datacite: Data
     InputState(label, description, timestamp)
   }
 
+  private def getOrAskEnumProp(enum: Enumeration, propertyKey: FeedBackMessage, propertyDescription: FeedBackMessage, props: PropertiesConfiguration, depositId: DepositId): enum.Value = {
```

Comment:
are there only two usages of this extraction, or could more of them be extracted? Looks like a very and reusable piece of code!

Reply:
In this source yes. Another pull for other instances.